doc/BUILD-Debian.md

```diff
@@ -0,0 +1,74 @@
+1. install boost (apt-get)
+2. install NVidia drivers
+3. install CUDA
+4. install ccmake
+5. install OpenCV
+6. install eigen
+7. install jpeg9a
+8. install libpng
+9. install ncurses
```

Comment:
instead of this `libtbb-dev`  (v4.0)

Reply:
Agreed, ncurses is required for ccmake, not for CCTag.